apps/src/sharedFunctionalBlocks.js

```diff
@@ -31,7 +31,9 @@ function installPlus(blockly, generator, gensym) {
       blockly.FunctionalBlockUtils.initTitledFunctionalBlock(this, '+', blockly.BlockValueType.NUMBER, [
         { name: 'ARG1', type: blockly.BlockValueType.NUMBER },
         { name: 'ARG2', type: blockly.BlockValueType.NUMBER }
-      ]);
+      ], { titleFontSize: 25});
```

Comment:
Are you just suggesting I have a variable like arithmeticTitleFontSize instead of specifying 25 in four places or something else?